ciao-launcher/main.go

```diff
@@ -563,7 +564,7 @@ DONE:
 			glog.Flush()
 
 			/* We panic here to see which naughty go routines are still running. */
-
+			debug.SetTraceback("all")
```

Comment:
Hidden away in a dark corner of the launcher docs we have this.

https://github.com/01org/ciao/blob/master/ciao-launcher/doc.go#L47

Could you just delete those two sentences.